### DISCUSSION TITLE
Do I have to bind the `<Plug>RestNvim` to a shortcut to have trigger the call?

### DISCUSSION THREAD

**Discussion by LintaoAmons** [rest-nvim/rest.nvim#282] [Q&A]
Because I actually want to type the command in command mode to trigger the call, but it seems that the `<plug>RestNvim` wont work if I type `:<Plug>RestNvim<CR>`

**Comment by NTBBloodbath** [comment]
Hi, that's right. `rest.nvim` is currently using `<Plug>` mappings which according to the docs (`:h using-<Plug>`) are used to avoid that mappings of typed keys interfere with mappings that are only to be used from other mappings.

The only way of using it is through user-defined mappings. That is going to change in the upcoming version tho :)

**Reply by LintaoAmons** [reply]
Got, please add some commands to use~

**Reply by NTBBloodbath** [reply]
Oh yup, there will be a `:Rest` command in the upcoming version!

See the [Rest command](https://amartin.codeberg.page/posts/first-look-at-thunder-rest/#rest-command) section of my blog post for more information :)

**Comment by boltlessengineer** [comment]
Closing this discuession as rest.nvim now have `:Rest` command for general operations.
`<plug>` is deprecated and removed from v3 release.
